team/tech/coordination.md

```diff
@@ -2,6 +2,22 @@
 
 As a totally distributed team, it is important to have team practices that give one another insight into what we're working on, identify opportunities to collaborate, and un-block one another. This page describes some of the practices that we adopt towards these goals.
 
+
+(coordination:team-syncs)=
+## Weekly team syncs
+
+Every Monday, the 2i2c Hub Operations team conducts a team sync to get on the same page and coordinate their work for the week.
```

Comment:
Weekly cadence is good IMHO.
Two weeks is actually a long time for sync, more than one iteration per week is noisy and, most of the time, it does not add enough value.